osi_common.proto

```diff
@@ -395,4 +395,19 @@ message BaseMoving
     // The polygon is defined counter-clockwise.
     //
     repeated Vector2d base_polygon = 7;
+    
+    // The relative orientation acceleration of the moving object w.r.t. its parent
+    // frame and parent orientation acceleration in the center point of the bounding box
+    // (origin of the bounding box frame), noted in the parent frame.
+    // The orientation becomes global/absolute if the parent frame is inertial
+    // (all parent frames up to ground truth).
+    //
+    // <tt>Rotation_yaw_pitch_roll(#orientation (t)) :=
+    // Rotation_yaw_pitch_roll(#orientation_rate
```

Comment:
Please provide an example for orientation_acceleration here.
Moreover, the last multiply should be a plus?
That seems to be wrong in the documentation of orientation rate...